app/models/journeyDomain/routeDetails/locationOfGoods/LocationOfGoodsDomain.scala

```diff
@@ -31,6 +31,8 @@ sealed trait LocationOfGoodsDomain extends JourneyDomainModel {
 
   val qualifierOfIdentification: LocationOfGoodsIdentification
 
+  val contactPerson: Option[AdditionalContactDomain] = None
```

Comment:
Could just call this `additionalContact` then override it where needed inside the case class signature